dash/_callback.py

```diff
@@ -342,7 +344,7 @@ def register_callback(
         flat_state,
         inputs_state_indices,
         prevent_initial_call,
-        long=long,
```

Comment:
@T4rk1n What about this argument called `long=` here, is there a reason it needs to stay?